plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/inner-blocks/taxonomy-filter/edit.tsx

```
@@ -0,0 +1,156 @@
+	options: FilterOptionItem[],
+	sortOrder: string
+): FilterOptionItem[] => {
+	return [ ...options ].sort( ( a, b ) => {
+		switch ( sortOrder ) {
+			case 'name-asc':
+				return a.value > b.value ? 1 : -1;
+			case 'name-desc':
+				return a.value < b.value ? 1 : -1;
+			case 'count-asc':
+				return a.count > b.count ? 1 : -1;
+			case 'count-desc':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)